code/go/0chain.net/blobbercore/blobbergrpc/proto/blobber.proto

```diff
@@ -66,7 +66,6 @@ service Blobber {
       }
       additional_bindings: {
         delete: "/v2/file/upload/{allocation}"
-        body: "*"
       }
```

Comment:
There is a delete api that accepts data in its body, you should handle this in a separate issue

Reply:
It is not possible, DELETE request body is explicitly discarded by the HTTP protocol.

Reply:
FYI. where it is written, please read this :
```
The spec does not explicitly forbid or discourage it, so I would tend to say it is allowed.

Microsoft sees it the same way (I can hear murmuring in the audience), they state in the MSDN article about the DELETE Method of ADO.NET Data Services Framework:

If a DELETE request includes an entity body, the body is ignored [...]
Additionally here is what RFC2616 (HTTP 1.1) has to say in regard to requests:

an entity-body is only present when a message-body is present (section 7.2)
the presence of a message-body is signaled by the inclusion of a Content-Length or Transfer-Encoding header (section 4.3)
a message-body must not be included when the specification of the request method does not allow sending an entity-body (section 4.3)
an entity-body is explicitly forbidden in TRACE requests only, all other request types are unrestricted (section 9, and 9.8 specifically)
For responses, this has been defined:

whether a message-body is included depends on both request method and response status (section 4.3)
a message-body is explicitly forbidden in responses to HEAD requests (section 9, and 9.4 specifically)
a message-body is explicitly forbidden in 1xx (informational), 204 (no content), and 304 (not modified) responses (section 4.3)
all other responses include a message-body, though it may be of zero length (section 4.3)
```

However, not going into much details this issue should be separately handled.